client/src/island/hexalot.ts

```diff
@@ -17,7 +17,9 @@ interface IHexalotIndexed {
 
 const spotsToHexFingerprint = (spots: Spot[]): string => {
     const lit = spots.map(spot => spot.surface === Surface.Land ? "1" : "0")
-    const nybbleStrings = lit.map((l, index, array) => (index % 4 === 0) ? array.slice(index, index + 4).join("") : null).filter(chunk => chunk)
+    const nybbleStrings = lit
+        .map((l, index, array) => (index % 4 === 0) ? array.slice(index, index + 4).join("") : null)
```

Comment:
Weird formatting

Reply:
Suggest an alternative (ctrl-g) and I'll accept